lib/membrane/simple_rtsp_server/pipeline.ex

```diff
@@ -99,7 +95,11 @@ defmodule Membrane.SimpleRTSPServer.Pipeline do
         clock_rate: config.clock_rate
       ]
     )
-    |> child({:realtimer, type}, Membrane.Realtimer)
+    |> then(
+      &if realtime,
```

Comment:
do you have a use case for realtime: false? I think it would overload the socket buffer and cause packet loss

Reply:
I wanted to have the test not take so long, but came to the same conclusion

Reply:
An alternative would be an option in realtimer that would speed it up a few times. And a relatively short fixture - I'm using up to 10s for such cases